scripts/update-config-fixtures.ts

```diff
@@ -0,0 +1,36 @@
+import { readFileSync, writeFileSync } from 'fs';
```

Comment:
My concern here is that we match and rewrite some files we shouldn't. TBH I imagined this script to contain a list of files we know need to be changed without any search or anything. I guess this is fine but I would at least add a warning saying to check git that all the modified files should really be modified. Or print a list of modified files.

Reply:
It prints the modified files (see the PR description)